utils.js

```diff
@@ -904,6 +928,39 @@ const removeDomainFromCloudFrontDistribution = async (cf, subdomain) => {
   }
 }
 
+const triggerTypes = ['viewer-request', 'origin-request', 'origin-response', 'viewer-response']
+
+const createLambda = async (lambda, trigger, subdomain, role) => {
+  const publishedLambda = await lambda({
+    runtime: 'nodejs10.x',
```

Comment:
Should we use `12.x`?

Reply:
Good idea. Let me quickly make changes.

Reply:
I tried to publish but this is Lambda@Edge and it only supports following runtime environments nodejs8.10, nodejs10.x, nodejs6.10, python3.7

Reply:
ah sad!